package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "challenge-frontend-150",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-scripts": "3.2.0",
+    "axios": "^0.19.0"
+  },
+  "devDependencies": {
+    "@igloosoftware/eslint-config-igloo": "^2.3.1",
+    "@igloosoftware/stylelint-config-igloo": "latest",
+    "eslint": "^6.6.0",
+    "stylelint": "^9.5.0",
+    "stylelint-order": "^1.0.0",
+    "stylelint-scss": "^3.3.0"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
```

Add purposefully undocumented script `lint` 👍 
```suggestion
    "test": "react-scripts test",
    "lint": "eslint .",
```